using_images/other_images/jenkins.adoc

```diff
@@ -64,6 +79,71 @@ $ oc new-app -e \
     openshift/jenkins-1-centos7
 ----
 
+If `*JENKINS_PASSWORD*` is not specified on the `oc new-app` invocation, the password is an auto-generated, random string.  It will be reported in the new-app
+output.  If you lose this value you can retrieve it by looking at the environment variable values defined on the Jenkins deployment configuration in the web console or by
+    running:
+
+        $ oc env dc/jenkins --list | grep JENKINS_PASSWORD
+
+ifdef::openshift-origin[]
+==== {product-title} OAuth authentication
+
+{product-title} OAuth authentication will be used when it is explicitly configured in the `*Configure Global Security*` panel, or it will be used automatically for the {product-title} Jenkins images
+when OpenShift Login plugin determines that it is running in a pod and the `*OPENSHIFT_ENABLE_OAUTH*` environment variable on the Jenkins `*Deployment Config*` is set to anything other than `false`.  The plugin will retrieve the configuration information needed
+either from data seeded into the pod, or by interacting with the {product-title} API server via REST invocations.  There are configuration fields available for the plugin, but they should only be required for perhaps
+development testing or atypical configurations when you want to access a different {product-title} installation for OAuth than the installation hosting Jenkins.
+
+The login process is comprised of a series of HTTP redirects.  Rather than the traditional Jenkins login page, you will first be directed to an {product-title} login page, similar to what you see when you log into the web console.  If login
+is successful, you will be redirected to the page within the Jenkins UI you are trying to access.
+
+Valid user / password combinations are controlled by the identity provider in use with {product-title}.  So, for example,  if the default identity provider `Allow All` is used, you can provide any
+non-empty string as the password for `admin` or any other user name of your choosing.  If one of the other identity providers is running, valid user/password combinations stored with your identity provider must be used to login.
```

Comment:
ok ... i'll have to revisit one aspect of my testing before i agree - with the requirement of using the self-sar to get permissions, it thought the user that created the project did not have the admin role with the OOTB `Allow All` scenario; if correct, then I think this impacts the OOTB experience, as the developer won't have enough power in Jenkins to do stuff.

I'll circle back and report findings once I get to try it, and we'll go from there.